c7n/resources/account.py

```diff
@@ -367,6 +367,52 @@ def process(self, resources, event=None):
         return []
 
 
+@filters.register('access-analyzer')
+class AccessAnalyzer(Filter):
+    """Check for access analyzers in an account
+
+    :example:
+
+    .. code-block:: yaml
+
+            policies:
+              - name: account-access-analyzer
+                resource: account
+                region: us-east-1
+                filters:
+                  - type: access-analyzer
+                    status: active
+                    trust: organization
+    """
+
+    schema = type_schema('access-analyzer',
+                        status={'oneOf': [{'enum': ['active', 'creating', 'disabled', 'failed']}]},
+                        trust={'oneOf': [{'enum': ['account', 'organization']}]})
+    schema_alias = False
+    permissions = ('access-analyzer:ListAnalyzers',)
+
+    def process(self, resources, event=None):
+        account = resources[0]
+        status = self.data.get('status', 'active')
+        trust = self.data.get('trust', '')
+
+        client = local_session(self.manager.session_factory).client('accessanalyzer')
+
+        try:
+            analyzers = self.manager.retry(client.list_analyzers)['analyzers']
+            for analyzer in analyzers:
+                if (not trust and analyzer['status'].lower() == status) or \
+                   (analyzer['type'].lower() == trust and analyzer['status'].lower()):
+                    account['c7n:AccessAnalyzer'] = analyzer
```

Comment:
there's quite a few attributes on the analyzer, i think its probably better for this filter to be value filter derived so that the user can specify the fields of interest to them.

Reply:
I think I managed to convert it to use ValueFilter, it's just that the policies are kinda weird because there are multiple analyzers so the key has to be prefixed with '[]' and the value is case sensitive. Not sure if there's a way around this or my implementation isn't optimal.